extensions/amp-inline-gallery/1.0/amp-inline-gallery-thumbnails.js

```diff
@@ -1,16 +1,16 @@
-import {CSS as CAROUSEL_CSS} from '../../amp-base-carousel/1.0/component.jss';
-import {CarouselContextProp} from '../../amp-base-carousel/1.0/carousel-props';
-import {PreactBaseElement} from '#preact/base-element';
-import {CSS as THUMBNAIL_CSS} from './thumbnails.jss';
-import {BentoInlineGalleryThumbnails} from './thumbnails';
 import {isExperimentOn} from '#experiments';
 import {userAssert} from '#utils/log';
+import {
+  TAG as BENTO_TAG,
+  ThumbnailsBaseElement,
+} from './thumbnails-base-element';
+import {getAmpName} from './utils';
 
 /** @const {string} */
-export const TAG = 'amp-inline-gallery-thumbnails';
+const TAG = getAmpName(BENTO_TAG);
+export {TAG};
```

Comment:
Is this how you're getting around the `no-export-side-effects` lint rule? 🙃

My 2c is that it might be better if we hardcode the tag name in both implementations.

Reply:
Yeahh i was confused why it was a considered a side effect in the first place, but yeah I can just hard code it